adele-booking/src/test/java/com/microservicesteam/adele/booking/domain/BookingIdGeneratorTest.java

```diff
@@ -0,0 +1,24 @@
+package com.microservicesteam.adele.booking.domain;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class BookingIdGeneratorTest {
+
+    private BookingIdGenerator bookingIdGenerator;
+
+    @Before
+    public void setUp() throws Exception {
+        bookingIdGenerator = new BookingIdGenerator();
+    }
+
+    @Test
+    public void bookingIdIsRandom() throws Exception {
```

Comment:
Nitpicking here, but I think this test is about the uniqueness of the id and not about it's randomness :)